### PR TITLE
PRT-286: Refactor nits

### PR DESCRIPTION
**Pull request by mauricedesaxe**
- PascalCase for PoolSymbol type
- uppercased the contract symbols in docs
- fixed ts-ignores caused by `new ethers.Contract`
- removed unused vars

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -320,15 +320,15 @@ export class ToucanClient {
   /**
    *
    * @description gets the remaining space in pool contract before hitting the cap
-   * @param poolSymbol symbol of the token to use
+   * @param PoolSymbol symbol of the token to use
    * @returns BigNumber representing the remaining space
    */
-  getPoolRemaining = async (poolSymbol: poolSymbol): Promise<BigNumber> => {
+  getPoolRemaining = async (PoolSymbol: PoolSymbol): Promise<BigNumber> => {
```
Comment (pheuberger) [inline index.ts:326]: the variable name should remain camel case, though

Comment (pheuberger) [reply index.ts:326]: i would also call it `pool` like you did on line 342

File: index.ts [hunk 2]
```diff
@@ -692,23 +692,23 @@ export class ToucanClient {
...
   /**
    *
    * @dev
    * @description gets the contract of a pool token based on the symbol
-   * @param poolSymbol symbol of the pool (token) to use
+   * @param PoolSymbol symbol of the pool (token) to use
    * @returns a ethers.contract to interact with the pool
    */
-  public getPoolContract = (poolSymbol: poolSymbol): IToucanPoolToken => {
+  public getPoolContract = (PoolSymbol: PoolSymbol): IToucanPoolToken => {
```
Comment (pheuberger) [inline index.ts:706]: see comment above

File: subclasses/ContractInteractions.ts [hunk 3]
```diff
@@ -346,15 +346,15 @@ class ContractInteractions {
...
    *
    * @description gets the remaining space in pool contract before hitting the cap
-   * @param poolSymbol symbol of the token to use
+   * @param PoolSymbol symbol of the token to use
    * @param signerOrProvider this being a read transaction, we need a signer or provider
    * @returns BigNumber representing the remaining space
    */
   getPoolRemaining = async (
-    poolSymbol: poolSymbol,
+    PoolSymbol: PoolSymbol,
```
Comment (pheuberger) [inline subclasses/ContractInteractions.ts:354]: see comment above

File: subclasses/ContractInteractions.ts [hunk 4]
```diff
@@ -548,24 +548,24 @@ class ContractInteractions {
...
    * @dev
    * @description gets the contract of a pool token based on the symbol
-   * @param poolSymbol symbol of the pool (token) to use
+   * @param PoolSymbol symbol of the pool (token) to use
    * @param signerOrProvider depending on what you intend to do with the contract, a signer or provider
    * @returns a ethers.contract to interact with the pool
    */
   public getPoolContract = (
-    poolSymbol: poolSymbol,
+    PoolSymbol: PoolSymbol,
```
Comment (pheuberger) [inline subclasses/ContractInteractions.ts:564]: see comment above

File: types/methods.ts [hunk 5]
```diff
@@ -155,7 +155,7 @@ export type fetchPoolContentsResult = Pick<PooledTCO2TokenSchema, "amount"> & {
 };
 
 export type fetchPoolContentsMethod = (
-  poolSymbol: poolSymbol,
+  PoolSymbol: PoolSymbol,
```
Comment (pheuberger) [inline types/methods.ts:158]: see comment above

File: test/index.ts [hunk 6]
```diff
@@ -132,12 +132,11 @@ describe("Testing Toucan-SDK", function () {
     });
 
     it("Should automatically redeem NCT & deposit the TCO2 back", async function () {
-      // @ts-ignore
       const tco2: IToucanCarbonOffsets = new ethers.Contract(
```
Comment (pheuberger) [inline test/index.ts:133]: if you use `as XYZ` you don't need to explicitly declare the type here. `const tco2 = new ...` is enough

**Review by pheuberger — DISMISSED** [review]

**Review by kargakis — APPROVED** [review]

**Pull request merged**